shared/js/background/script-injection.js

```diff
@@ -0,0 +1,25 @@
+const browserWrapper = require('./wrapper.es6')
+
+// With Manifest V2 the content scripts are run directly from the manifest and
+// they inject into the main world of websites by adding a <script> element to
+// the DOM. With Manifest V3 however the chrome.scripting API must be used to
+// inject scripts into the main world.
+if (browserWrapper.getManifestVersion() === 3) {
+    // @ts-ignore - The chrome type does not yet know about
```

Comment:
I'll merge https://github.com/duckduckgo/duckduckgo-privacy-extension/pull/1391 now I think this might be fixed.

Reply:
Dang, just checked but it's still not in there :(.

Reply:
:( looks like they're working on that area though from the quick scan.